### PR TITLE
CRYPTO-34: JavaSecureRandom should throw Exception if it cannot create the instance

### PR DESCRIPTION
**Pull request by sundapeng**
Fixing https://issues.apache.org/jira/browse/CRYPTO-34

### CODE REVIEW

File: src/test/java/org/apache/commons/crypto/random/AbstractRandomTest.java [hunk 1]
```diff
@@ -17,14 +17,13 @@
...
 
-import java.io.IOException;
 import java.util.Arrays;
 
 import org.junit.Test;
 
 public abstract class AbstractRandomTest {
 
-  public abstract SecureRandom getSecureRandom() throws IOException;
+  public abstract SecureRandom getSecureRandom() throws Exception;
```
Comment (jerrychenhf) [inline src/test/java/org/apache/commons/crypto/random/AbstractRandomTest.java]: May GeneralSecurityException here?

**Comment by jerrychenhf** [comment]
Thanks @sundapeng for contributing. Leave a minor comment.

**Comment by sundapeng** [comment]
Thank @jerrychenhf for your review, I have updated the patch

**Comment by winningsix** [comment]
+1

**Comment by sundapeng** [comment]
Thank @jerrychenhf and @winningsix for your review

**Pull request merged**